cmd/ci-secret-mirroring/README.md

```diff
@@ -0,0 +1,56 @@
+# CI-Secret-Mirroring
+
+This tool extends the [populate-secrets-from-bitwarden.sh](https://github.com/openshift/release/blob/c8c89d08c56c653b91eb8c7580657f7ce522253f/ci-operator/populate-secrets-from-bitwarden.sh)
+to support mirroring secrets cross Kubernetes/OpenShift-clusters.
+
+We use `--kubeconfig` to specify the path to a [kube config](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
+that the tool will load and use it to access clusters for writing secrets.
+
+It expects a configuration like the one below which specifies the mapping from the items
+in BitWarden and the targeting secret.
+
+```yaml
+secrets:
+- from:
+    - bw-item: item-name-1
```

Comment:
Would it ever make sense to have more than one item mapped to a key name?

Reply:
If yes, when? If not, what other data structures could more efficiently hold this mapping?